docs/wifi.md

```diff
@@ -0,0 +1,78 @@
+---
+sidebar_label: Configure Wi-Fi
+title: ''
+---
+
+
+### How to configure Wi-Fi
+
+:::info info
+The information on this page is just a specific use case of using cloud-config. For more generic info on how to create arbitrary files check our [cloud-config](cloud-config-reference.md) page
+:::
+
+Elemental Teal currently uses [NetworkManager](https://networkmanager.dev/) to manage network connections.
+
+In order to add Wi-Fi to your node, your registration should include a configuration in the [cloud-config](cloud-config-reference.md) section to write a
+`.connection` file so NetworkManager can connect to the Wi-Fi.
+
+A `.connection` file is a connection configuration file for NetworkManager.
+The connection files are stored under `/etc/NetworkManager/system-connections/` and can include ethernet, Wi-Fi, VPN and more.
+
+For example for a network with the SSID `testSSID` and the WEP key `1234567891` and using the interface `wlan0` you would write the following `.connection` file:
+
+```
+[connection]
+id=testConnection
+type=wifi
+interface-name=wlan0
+permissions=
+timestamp=1671549641
+
+[wifi]
+mac-address-blacklist=
+mode=infrastructure
+ssid=testSSID
+
+[wifi-security]
+key-mgmt=none
+wep-key-type=1
+wep-key0=1234567891
```

Comment:
What about a WPA template instead of WEP?
Something like:
```
[wifi-security]
key-mgmt=wpa-psk
psk=1234567891
```

Reply:
Sure, no probs there